admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
Ah I see, yes that sounds good! This way we don't need `is_level_admin()` at all.
I think it can just be put inside the current filter method since it always runs.

> You can see that on multisites, the function is already returned before reaching the user_has_cap filter, so there shouldn't be a need to check for that.

Keep in mind that for non-super admins this filter is still used.

See my latest commit.
I chose to always add the capability (not just when the requested cap is the same). This makes sure we're always returning the same data for clearity to other plugin filters.

What shall we do with the `is_level_admin()` method and other is_super_admin checks I've built in there? Do you want to remove it?